mpas_analysis/sea_ice/climatology_map_melting.py

```diff
@@ -0,0 +1,330 @@
+        climatology : xarray.Dataset
+            the climatology data set
+        season : str
+            The name of the season to be masked
+        Returns
+        -------
+        climatology : xarray.Dataset
+            the modified climatology data set
+        """
+
+        melting = self._compute_melting(climatology)
+
+        climatology['seaIceMelting'] = melting
+        climatology.seaIceMelting.attrs['units'] = 'm y^-1'
+        climatology = climatology.drop_vars(self.variable_list)
+
+        return climatology
+
+    def _compute_melting(self, climatology):
+        """
+        Compute the total sea ice melting in m yr^-1 from the individual
+        melt fields in m s^-1.
+        """
+        ds_restart = xr.open_dataset(self.restartFileName)
+        ds_restart = ds_restart.isel(Time=0)
+
+        units_scale_factor = 60 * 60 * 24 * 365
+
+        basal = climatology['timeMonthly_avg_basalIceMelt']
+        surface = climatology['timeMonthly_avg_surfaceIceMelt']
+        lateral = climatology['timeMonthly_avg_lateralIceMelt']
+
+        melting = (basal + surface + lateral) * units_scale_factor
+        return melting
+
+class RemapHaumannMeltingClimatology(RemapObservedClimatologySubtask):
+    """
+    A subtask for reading and remapping sea ice melting from Haumann
+    observations
+    """
+    # Authors
+    # -------
+    # Darin Comeau, Xylar Asay-Davis
+
+    def get_observation_descriptor(self, fileName):
+        """
+        get a MeshDescriptor for the observation grid
+
+        Parameters
+        ----------
+        fileName : str
+            observation file name describing the source grid
+
+        Returns
+        -------
+        obsDescriptor : ``MeshDescriptor``
+            The descriptor for the observation grid
+        """
+        # Authors
+        # -------
+        # Darin Comeau, Xylar Asay-Davis
+
+        # create a descriptor of the observation grid using the lat/lon
+        # coordinates
+        obsDescriptor = LatLon2DGridDescriptor.read(fileName=fileName,
+                                                    latVarName='lat',
+                                                    lonVarName='lon')
+        return obsDescriptor
+
+    def build_observational_dataset(self, fileName):
+        """
+        read in the data sets for observations, and possibly rename some
+        variables and dimensions
+
+        Parameters
+        ----------
+        fileName : str
+            observation file name
+
+        Returns
+        -------
+        dsObs : ``xarray.Dataset``
+            The observational dataset
+        """
+        # Authors
+        # -------
+        # Darin Comeau, Xylar Asay-Davis
+
+        dsObs = xr.open_dataset(fileName)
+        dsObs = dsObs.rename({'melting': 'seaIceMelting', 'time': 'Time'})
+        # dsObs.coords['month'] = dsObs['Time.month']
+        # dsObs.coords['year'] = dsObs['Time.year']
+        dsObs = dsObs.transpose('Time', 'y', 'x')
```

Comment:
This seems to be working for me. I think you want `Time` in there if you're going to compute a climatology and you don't if the climatology has already been computed (so `year` and `months` also aren't there).
```suggestion
        dsObs = dsObs.isel(time=0)
        dsObs = dsObs.rename({'melting': 'seaIceMelting'})
        dsObs = dsObs.transpose('y', 'x')
```